.github/ISSUE_TEMPLATE/documentation-issue.md

```diff
@@ -0,0 +1,12 @@
+---
+name: Documentation Issue
+about: Changes to the documentation.
```

Comment:
See my other note for readme.md. 

Why not refer to https://github.com/camunda/camunda-platform-docs?

Reply:
Agreed, I created a link to the camunda platform docs repository instead 👍